src/System.Management.Automation/engine/Modules/NewModuleManifestCommand.cs

```diff
@@ -1197,6 +1217,9 @@ private void BuildPrivateDataInModuleManifest(StringBuilder result, StreamWriter
                 BuildModuleManifest(result, "ProjectUri", Modules.ProjectUri, ProjectUri != null, () => QuoteName(ProjectUri), streamWriter);
                 BuildModuleManifest(result, "IconUri", Modules.IconUri, IconUri != null, () => QuoteName(IconUri), streamWriter);
                 BuildModuleManifest(result, "ReleaseNotes", Modules.ReleaseNotes, !string.IsNullOrEmpty(ReleaseNotes), () => QuoteName(ReleaseNotes), streamWriter);
+                BuildModuleManifest(result, "Prerelease", Modules.Prerelease, !string.IsNullOrEmpty(Prerelease), () => QuoteName(Prerelease), streamWriter);
+                BuildModuleManifest(result, "RequireLicenseAcceptance", Modules.RequireLicenseAcceptance, RequireLicenseAcceptance.IsPresent, () => { return RequireLicenseAcceptance.IsPresent ? "$true" : "False"; }, streamWriter);
```

Comment:
```suggestion
                BuildModuleManifest(result, "RequireLicenseAcceptance", Modules.RequireLicenseAcceptance, RequireLicenseAcceptance.IsPresent, () => { return RequireLicenseAcceptance.IsPresent ? "$true" : "$False"; }, streamWriter);
```

Reply:
I agree with what you are suggesting. Update-ModuleManifest specifies it as "False" and not "$False" which is why I didn't simply go with $false in the first place.

Reply:
I don't understand the difference. Could you please point the code or an example?